org/org.yaml

```diff
@@ -67,3 +67,4 @@ orgs:
     - vincent-pli
     - vtereso
     - wlynch
+    -spomorski
```

Comment:
minor nit: keep the alphabetical ordering & add a space after the `-`?

Reply:
Oops, fixed and fixed.